emptyStrCmp/emptyStrCmp.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package emptystrcmp
```

Comment:
The package name and the directory names are different. Let's make them same.

Reply:
go-lint says not to use mixed cap names for package declaration so I had kept it all in small case